From 2aa7964f9d31448fa0f3caa17bcbe032c5f63d4a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Wed, 24 Nov 2021 20:21:09 +0100
Subject: [PATCH] Fixup includes for VTK 9.1 compatibility

With VTK 9.1 vtkCommand.h no longer implicitly includes vtkVersion.h.

Remove vtkCommand.h from QVTKStructure.h, as it is not necessary, but
explicitly include vtkVersion.h.
---
 QVTKStructure.cpp | 2 +-
 QVTKStructure.h   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/QVTKStructure.cpp b/QVTKStructure.cpp
index 1a6dff5..c06fc7f 100644
--- a/QVTKStructure.cpp
+++ b/QVTKStructure.cpp
@@ -19,7 +19,6 @@
 
 #include "QVTKStructure.h"
 
-#include "vtkCommand.h"
 #if VTK_MAJOR_VERSION>=9
   #include "QVTKOpenGLStereoWidget.h"
   #include "vtkGenericOpenGLRenderWindow.h"
@@ -30,6 +29,7 @@
   #include "QVTKWidget.h"
 
 #endif
+#include "vtkCommand.h"
 #include "vtkRenderWindow.h"
 #include "vtkRenderWindowInteractor.h"
 #include "vtkRenderer.h"
diff --git a/QVTKStructure.h b/QVTKStructure.h
index 4b0dd5e..70f8bac 100644
--- a/QVTKStructure.h
+++ b/QVTKStructure.h
@@ -20,7 +20,7 @@
 
 #include <QtGui>
 
-#include "vtkCommand.h"
+#include "vtkVersion.h"
 #if VTK_MAJOR_VERSION>=9
   class QVTKOpenGLStereoWidget;
 #elif VTK_MAJOR_VERSION==8
-- 
2.33.1

